boot/pod/src/boot/tmpdir.clj

```diff
@@ -100,7 +100,11 @@
                    (add-blob! blob path i link)
                    (swap! tree assoc p (map->TmpFile (assoc m :path p :id i :hash h :time t))))
                  (catch java.nio.file.NoSuchFileException _
-                   (util/dbug* "Tmpdir: file not found: %s\n" (.toString p))))) )))))
+                   (util/dbug* "Tmpdir: file not found: %s\n" (.toString p)))))))
+      (visitFileFailed [^Path path ^java.io.IOException e]
+        (with-let [_ fs/skip-subtree]
+          (let [p (str (.relativize root path))]
+            (util/dbug* "Tmpdir: failed to visit: %s\n" (.toString p))))))))
```

Comment:
A minor thing: Now that .relativize call is outside of the dbug* macro call, it is evaluated always. If it was inside the macro call, if would be evaluated only when dbug output is enabled.

Also, it is unnecessary to have both str and .toString calls.